src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -93,8 +123,7 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
       </td>
 
       <td>
-        Correlation activated because the New Relic non-NRQL violation target entity names are the same. Does not apply to REST source. Non-NRQL entity refers to [entity](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/), typically APPLICATION, HOST types, see [New Relic GitHub repo on entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions). Relevant incidents from the same entity will be identified. For example, host high memory incidents and host not-reporting incidents could be highly possible due to the same cause.
-      </td>
+        Correlation activated because the New Relic non-NRQL violation target entity names are the same. Does not apply to REST source. Non-NRQL entity refers to [entity](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/), typically APPLICATION, HOST types, see [New Relic GitHub repo on entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions). Relevant issues from the same entity will be identified. For example, host high memory issue and host not-reporting issue could be highly possible due to the same cause.</td>
```

Comment:
this paragraph needs some review. second sentence is hard to parse and grammatically incorrect. last sentence is awkward (highly possible due to)